tensorflow/lite/micro/tools/make/targets/cortex_m_generic_makefile.inc

```diff
@@ -0,0 +1,156 @@
+# Generic Makefile target for ARM Cortex M builds.
+# REQUIRED:
+#   - TAGS: armclang or armgcc
+# For Cortex-M55, ARM Compiler 6.14 or later is required.
+# Example:
+# make -f tensorflow/lite/micro/tools/make/Makefile TAGS="cmsis-nn armclang"  TARGET=cortex-m55-generic microlite
+
+ifneq ($(filter cortex-%-generic,$(TARGET)),)
```

Comment:
With https://github.com/tensorflow/tensorflow/pull/43896 we are moving towards enforcing that the TARGET correspond to an actual directory. So we should go with `TARGET=cortex_m_generic` and have a separate command line option for the processor type (just like `CORTEX_M_CORE` in the previous version of the generic makefile).